packages/angular-test-app/src/preview-examples/toggle-button-secondary.ts

```diff
@@ -12,10 +12,10 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-example',
   template: `
-    <ix-button class="m-1" variant="secondary" ghost>Not selected</ix-button>
-    <ix-button class="m-1" variant="secondary" ghost selected
-      >Selected</ix-button
-    >
+    <ix-toggle-button class="m-1">Normal</ix-toggle-button>
+    <ix-toggle-button class="m-1" pressed> Pressed </ix-toggle-button>
+    <ix-toggle-button class="m-1" disabled> Disabled </ix-toggle-button>
+    <ix-toggle-button class="m-1" disabled loading> Loading </ix-toggle-button>
```

Comment:
If we imeplement/change examples should we still use bootstrap utility calsses like `m-1`?